UI/MultiIconValueIndicator.cpp

```diff
@@ -28,7 +28,7 @@ MultiIconValueIndicator::MultiIconValueIndicator(GG::X w) :
 
 MultiIconValueIndicator::MultiIconValueIndicator(GG::X w, int object_id,
                                                  const std::vector<std::pair<MeterType, MeterType>>& meter_types) :
-    MultiIconValueIndicator(w, {{object_id}}, meter_types)
+    MultiIconValueIndicator(w, std::vector<int>{object_id}, meter_types)
```

Comment:
Hm, even this is wrong.  The rules regarding initializer list and uniform initialization are confusing…

Your version seems to be correct.  If a class provides a constructor with initializer lists it will prefer this over the interpretation as uniform initialization if the initializer list contains valid initialization values for a vector.